caddy/caddymain/run.go

```diff
@@ -98,13 +99,24 @@ func Run() {
 	}
 
 	// Get Caddyfile input
-	caddyfile, err := caddy.LoadCaddyfile(serverType)
+	caddyfileinput, err := caddy.LoadCaddyfile(serverType)
 	if err != nil {
 		mustLogFatalf(err.Error())
 	}
 
+	if validate {
+		justValidate := true
+		_, err := caddy.ValidateAndExecuteDirectives(caddyfileinput, nil, justValidate)
+		if err != nil {
+			mustLogFatalf(err.Error())
+		}
+		log.Println("[INFO] Caddyfile Valid")
+		//Caddyfile is valid so exit without starting
```

Comment:
Same thing here about the comment spacing, but I don't think this comment is even necessary